lib/grandpa/message_handler.go

```diff
@@ -98,6 +105,13 @@ func (h *MessageHandler) handleNeighbourMessage(msg *NeighbourMessage) error {
 		return nil
 	}
 
+	// TODO: make linter british
```

Comment:
What does make linter british mean?

Reply:
lol, I always got linter issues for misspellings when spelling things Canadian (ie neighbour vs neighbor), apparently we can change the longer settings to use UK English. I'll open an issue for it